lib/actions/help.js

```diff
@@ -0,0 +1,23 @@
+const commands = require('../../config/commands.json')
```

Comment:
Maybe we can do this better later and somehow define this in the js files so the help list is created automatically